### DISCUSSION TITLE
Galaxy brain

### DISCUSSION THREAD

**Discussion by Crushoverride007** [Schweinepriester/github-profile-achievements#186] [Q&A]
Make sure to star my repo https://github.com/Crushoverride007/Crushoverride007 and i will give your the answer

**Comment by Luscarvalho** [comment]
ok, but Return the favor ;)

https://github.com/Luscarvalho/Luscarvalho

**Comment by Ysrbolles** [comment]
Make sure to star my repo https://github.com/Ysrbolles/Darkly-42Network
